src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1478,6 +1484,7 @@ PyObject *c_Node_to_py(const rd_kafka_Node_t *c_node) {
         Py_XDECREF(Node_type);
         Py_XDECREF(args);
         Py_XDECREF(kwargs);
+        Py_XDECREF(node);
```

Comment:
There's no case it can not-null and come here. Py_XDECREF can be removed, while what's missing is a condition after PyObject_Call that goes to err if node is NULL.

Reply:
I think doing this makes it future proof but I am fine with either way.